ic0/Cargo.toml

```diff
@@ -11,6 +11,7 @@ readme = "README.md"
 categories = ["api-bindings", "development-tools::ffi"]
 keywords = ["internet-computer", "types", "dfinity", "canister", "cdk"]
 include = ["src", "Cargo.toml", "LICENSE", "README.md"]
+links = "ic0"
```

Comment:
Why add links here?
IIUC, multiple versions of `ic0` can exist in a dependency tree since it doesn't set global state nor define exports.

Reply:
ah, I think I misunderstood its real purpose - you are correct